plugins/vpp/ifplugin/vppcalls/dump_nat_vppcalls.go

```diff
@@ -144,8 +147,47 @@ func (handler *natVppHandler) nat44AddressDump() (addresses []*nat.Nat44Global_A
 	return
 }
 
+// virtualReassemblyDump returns current NAT44 virtual-reassembly configuration. The output config may be nil.
+func (handler *natVppHandler) virtualReassemblyDump() (vrIPv4 *nat.Nat44Global_VirtualReassemblyIPv4, vrIPv6 *nat.Nat44Global_VirtualReassemblyIPv6, err error) {
+	defer func(t time.Time) {
+		handler.stopwatch.TimeLog(bin_api.NatGetReass{}).LogTimeEntry(time.Since(t))
+	}(time.Now())
+
+	req := &bin_api.NatGetReass{}
+	reqContext := handler.dumpChannel.SendRequest(req)
```

Comment:
Don't see how splitting `SendRequest` from `ReceiveReply` might be useful. Just one extra line with variable defined that is used once.